bin/generateGraphql.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
```

Comment:
What if this was also a bash file? Something like
echo -e "/* eslint:disable */\n// This file was automatically generated\nconst typeDefs = `\n$(cat ../api/graphql/schema-types-defs/*.graphql)\n`;\nmodule.exports = typeDefs;\n/* eslint:enable */" > ./api/graphql/__generated__/schema-defs.js

Reply:
Lovely...

Reply:
If could. doesn't have to be. any benefit of bash over js?

Reply:
Perhaps just conciseness, but the original intention was consistency. Your other bin script is a bash script, so I thought it might make sense to have them the same but, you're right, it doesn't have to be :)